src/structures/CommandInteractionOptionResolver.js

```diff
@@ -56,6 +54,14 @@ class CommandInteractionOptionResolver {
      */
     Object.defineProperty(this, 'data', { value: Object.freeze([...options]) });
 
+    /**
+     * The hoisted interaction options array, so without any subcommand(groups).
+     * @name CommandInteractionOptionResolver#hoistedOptions
+     * @type {ReadonlyArray<CommandInteractionOption>}
+     * @readonly
```

Comment:
I don't fully agree with this PR but this readonly tag is only for getters iirc

Reply:
@ImRodry - I pretty much copied the `data` property which is right above it. I wasn't sure how to document it otherwise

Reply:
`@readonly` is also used for `Object.defineProperty` too.